### ISSUE TITLE
Sentinel 1-RTC fringes until 2018

### ISSUE THREAD

**Issue by cordmaur** [microsoft/PlanetaryComputer#307]
I'm conducting a research using S1-RTC dataset and I noticed that some images have 0-filled borders (like fringes) that are not marked as "no-data" (examle below)

After 2018, this problem disappears, and that can be seen in the graph that counts for water...
Any idea on why is that and how to correctly get rid of those pixels? 

![image](https://github.com/microsoft/PlanetaryComputer/assets/19617404/e0659f38-dd28-49f6-a793-661da811d14c)

**Comment by TomAugspurger** [comment]
Thanks for the report. Could you share a sample STAC item ID that shows the problem?

**Comment by scottyhq** [comment]
@cordmaur the RTC products are generated from Sentinel-1 IW GRD products, and the treatment of border NaNs has changed over time.  ESA has a technical document explaining this in detail: [Masking "No-value" Pixels on GRD Products generated by the Sentinel-1 ESA IPF](https://sentinel.esa.int/web/sentinel/user-guides/sentinel-1-sar/document-library/-/asset_publisher/1dO7RF5fJMbd/content/masking-no-value-pixels-on-grd-products-generated-by-the-sentinel-1-esa-ipf?_com_liferay_asset_publisher_web_portlet_AssetPublisherPortlet_INSTANCE_1dO7RF5fJMbd_assetEntryId=2371063&_com_liferay_asset_publisher_web_portlet_AssetPublisherPortlet_INSTANCE_1dO7RF5fJMbd_redirect=https%3A%2F%2Fsentinel.esa.int%2Fweb%2Fsentinel%2Fuser-guides%2Fsentinel-1-sar%2Fdocument-library%3Fp_p_id%3Dcom_liferay_asset_publisher_web_portlet_AssetPublisherPortlet_INSTANCE_1dO7RF5fJMbd%26p_p_lifecycle%3D0%26p_p_state%3Dnormal%26p_p_mode%3Dview%26_com_liferay_asset_publisher_web_portlet_AssetPublisherPortlet_INSTANCE_1dO7RF5fJMbd_assetEntryId%3D2371063%26_com_liferay_asset_publisher_web_portlet_AssetPublisherPortlet_INSTANCE_1dO7RF5fJMbd_cur%3D0%26p_r_p_resetCur%3Dfalse). 

Long story short, you can follow the processing provenance to track down the IPF version of a GRD and there are various methods to clean up that border noise if you need to.

**Comment by ghidalgo3** [comment]
`Closed` due to inactivity, feel free to reopen if you would like to continue this discussion.
